spec.md

```diff
@@ -51,9 +51,12 @@ The [OCI Image Media Types](media-types.md) document is a starting point to unde
 
 The high-level components of the spec include:
 
-* An archival format for container images, consisting of an [image manifest](manifest.md), an [image index](image-index.md) (optional), an [image layout](image-layout.md), a set of [filesystem layers](layer.md), and [image configuration](config.md) (base OCI layer)
-* A [process of referencing container images by a cryptographic hash of their content](descriptor.md) (base OCI layer)
-* A format for [storing CAS blobs and references to them](image-layout.md) (optional OCI layer)
+* [Image Manifest](manifest.md) - the stored document describing an image's properties
+* [Image Index](image-index.md) - the stored document describing a collection of images redistributed as one
```

Comment:
More or less. Effectively, they reference one or more manifests, annotating the underlying content with platform information "pulled up" config. In practice, we could compare this to a "fat binary", although the content isn't embedded.

"An index of annotated image manifests distributed as a complete unit."

Not great, but that might help to better describe this.